src/main/resources/ApplicationInsights.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ApplicationInsights xmlns="http://schemas.microsoft.com/ApplicationInsights/2013/Settings" schemaVersion="2014-05-30">
+    <!-- The key from the portal: -->
+    <InstrumentationKey>d3278c59-22d7-4697-af8c-ff0077e3b4e9</InstrumentationKey>
```

Comment:
By exposing the key, will it render security risk?

Reply:
No security risk, this only used to send data. But if others may abuse this key. From spring-data-cosmosdb, it use placeholder here, and patch the real key when release (that means you can find the key from release package). But here we are web application different to release package. So I just put the key here.